go/exosecurity/data_requester.go

```diff
@@ -0,0 +1,6 @@
+package exosecurity
+
+import "github.com/Originate/exocom/go/structs"
+
+// DataRequester requests the data
```

Comment:
Lets make this more informative. Maybe something like 'sends a message and returns the first response'